pkg/server/api/bundle/v1/service.go

```diff
@@ -518,3 +569,50 @@ func applyBundleMask(b *types.Bundle, mask *types.BundleMask) {
 		b.JwtAuthorities = nil
 	}
 }
+
+func fieldsFromBundleProto(proto *types.Bundle, inputMask *types.BundleMask) logrus.Fields {
+	fields := logrus.Fields{
+		telemetry.TrustDomainID: proto.TrustDomain,
+	}
+
+	if inputMask == nil || inputMask.RefreshHint {
+		fields[telemetry.RefreshHint] = proto.RefreshHint
+	}
+
+	if inputMask == nil || inputMask.SequenceNumber {
+		fields[telemetry.SequenceNumber] = proto.SequenceNumber
+	}
+
+	if inputMask == nil || inputMask.JwtAuthorities {
+		for k, v := range fieldsFromJwtAuthoritiesProto(proto.JwtAuthorities) {
+			fields[k] = v
+		}
+	}
+
+	if inputMask == nil || inputMask.X509Authorities {
+		for k, v := range fieldsFromX509AuthoritiesProto(proto.X509Authorities) {
+			fields[k] = v
+		}
+	}
+	return fields
+}
+
+func fieldsFromJwtAuthoritiesProto(jwtAuthorities []*types.JWTKey) logrus.Fields {
+	fields := make(logrus.Fields, 3*len(jwtAuthorities))
+	for i, jwtAuthority := range jwtAuthorities {
+		fields[fmt.Sprintf("jwt_authority_expires_at.%d", i)] = jwtAuthority.ExpiresAt
+		fields[fmt.Sprintf("jwt_authority_key_id.%d", i)] = jwtAuthority.KeyId
+		fields[fmt.Sprintf("jwt_authority_public_key.%d", i)] = api.HashByte(jwtAuthority.PublicKey)
+	}
+
+	return fields
+}
+
+func fieldsFromX509AuthoritiesProto(x509Authorities []*types.X509Certificate) logrus.Fields {
+	fields := make(logrus.Fields, len(x509Authorities))
+	for i, x509Authority := range x509Authorities {
+		fields[fmt.Sprintf("x509_authorities_asn1.%d", i)] = api.HashByte(x509Authority.Asn1)
```

Comment:
1) Can we call this `x509_authorities_asn1_sha256` to indicate that this is a hash?
2) Can we use a constant from the telemetry package?